murageden/we-connect-v2-react-front-end
# #158820598 Redirect Users to Business Details Page after Adding a Review

#### What does this PR do?
Adds user redirection to the business details page after user adds a review
#### Description of Task to be completed?
* After user adds a review, the user should be directed to the business details page where they can see the review they have just added
#### How should this be manually tested?
* Set up the application as described on the `README`
* Switch to branch `ft-redirect-user-to-business-details-on-reviewing-158820598`
* Register an account and login, after reviewing a business, you should be directed to a page containing the business' reviews and view the review you have just created
#### Any background context you want to provide?
Previously a user was being directed to the home page
#### What are the relevant pivotal tracker stories?
Pivotal Tracker: https://www.pivotaltracker.com/n/projects/2177751
Story ID: 158820598
#### Questions:
How can this be improved?
